lighthouse/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y \
+  apt-transport-https \
+  ca-certificates \
+  curl \
+  gnupg \
+  --no-install-recommends \
+  && curl -sSL https://deb.nodesource.com/setup_12.x | bash - \
+  && curl -sSL https://dl.google.com/linux/linux_signing_key.pub | apt-key add - \
+  && echo "deb https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list \
+  && apt-get update && apt-get install -y \
+  google-chrome-stable \
+  fontconfig \
+  fonts-ipafont-gothic \
+  fonts-wqy-zenhei \
+  fonts-thai-tlwg \
+  fonts-kacst \
+  fonts-symbola \
+  fonts-noto \
+  fonts-freefont-ttf \
+  nodejs \
+  --no-install-recommends \
+  && apt-get purge --auto-remove -y curl gnupg \
+  && rm -rf /var/lib/apt/lists/*
+
+
+# install lighthouse
+RUN npm install -g @lhci/cli@0.6.x
```

Comment:
**nitpick:** Do we need to specify the version of LH CLI, by default it should just pull the latest version, which is 0.6.0 - https://www.npmjs.com/package/@lhci/cli

Reply:
We should maybe at least version lock the major version in case a new major version isn't compatible with our config

Reply:
Makes sense, thanks!